### PR TITLE
Prepare for release v1.2.8

### PR DESCRIPTION
**Pull request by ianvernon**
Signed-off by: Ian Vernon <ian@cilium.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/7133)

### CODE REVIEW

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
test-missed-k8s

**Comment by ianvernon** [comment]
test-docs-please

File: NEWS.rst [hunk 1]
```diff
@@ -2,6 +2,45 @@
 NEWS
 ******
 
+1.2.8
+=====
+
+::
```
Comment (aanm) [inline NEWS.rst:8]: Might be worth pointing out somewhere this is the last release of v1.2

**Review by aanm — APPROVED** [review]

File: NEWS.rst (hunk 1 above)
Comment (tgraf) [reply NEWS.rst:8]: I think we can mention that in the slack announcement

**Review by tgraf — APPROVED** [review]

**Pull request merged**